app/Http/Middleware/ApiAuthenticate.php

```
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Illuminate\Auth\Middleware\Authenticate as Middleware;
+use Illuminate\Http\Request;
+
+class ApiAuthenticate extends Middleware
+{
+    protected function unauthenticated($request, array $guards)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (Unused Code Rules)

(UnusedFormalParameter)